Gopkg.lock

```diff
@@ -264,7 +264,7 @@
     "pkg/server",
     "pkg/util"
   ]
-  revision = "aa344ee5f4fff239fadab634413ca6764025511c"
+  revision = "9fdf4bd79736a87c78b3d48b7baa84ea24be1c1a"
```

Comment:
Is this to much trouble to ask for adding /envoy/config/filter/http/jwt_authn/v2alpha in this PR as well. It would save me some time :)